integration/experiment/system_characterization/platform_sweeps.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+#  Copyright (c) 2015 - 2025 Intel Corporation
+#  SPDX-License-Identifier: BSD-3-Clause
+
+# This script is the launcher for platform power & frequency sweeps
+# for node characterization
+
+
+export SWEEP_TYPE="${SWEEP_TYPE?- Set SWEEP_TYPE to either CPU or GPU}"
```

Comment:
```suggestion
export SWEEP_TYPE="${SWEEP_TYPE:?Set SWEEP_TYPE to either CPU or GPU}"
```
https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html